exatomic/gaussian/output.py

```diff
@@ -84,14 +85,17 @@ def _parse_triangular_matrix(self, regex, column='coef', values_only=False):
                                       'frame': idxs[:,2],
                                        column: matrix})
 
-    def parse_atom(self):
+    def parse_atom(self, std_or=True):
```

Comment:
Could we make the kwarg here have values of regeom01/2 instead?

Reply:
I think we could do that. Since the parser is looking for `'Input orientation'` and `'Standard orientation'` we could have the keyword be `None`, `'Input'`, or `'Standard'`. Is that what you mean?

Reply:
Yeah (maybe default 'Input orientation' with allowable 'Standard orientation')